etc/contracts-test-data/contracts/loadnext/query_event_metrics.sql

```diff
@@ -0,0 +1,23 @@
+-- calculate distribution of event emissions per transaction
+
+\set :start_from_miniblock_number 40000000
+\set :miniblock_range 10000
+
+select stddev_samp(metric)                                  as stddev,
+       avg(metric)                                          as avg,
+       sum(metric)                                          as sum,
+       percentile_cont(0.00) within group (order by metric) as pct_00,
+       percentile_cont(0.01) within group (order by metric) as pct_01,
+       percentile_cont(0.05) within group (order by metric) as pct_05,
+       percentile_cont(0.25) within group (order by metric) as pct_25,
+       percentile_cont(0.50) within group (order by metric) as pct_50,
+       percentile_cont(0.75) within group (order by metric) as pct_75,
+       percentile_cont(0.95) within group (order by metric) as pct_95,
+       percentile_cont(0.99) within group (order by metric) as pct_99,
+       percentile_cont(1.00) within group (order by metric) as pct_100
```

Comment:
nit: I would just do `min`,`stdev`,`avg`, `max`, `median` - if someone's interested in more percentiles they can copypaste - but feel free to leave as is